src/plugins/drawing/components/drawing-layer.tsx

```diff
@@ -70,13 +71,21 @@ export class DrawingLayerView extends React.Component<DrawingLayerViewProps, Dra
       }
     });
 
+    this.viewRef = React.createRef();
     this.svgRef = null;
     this.setSvgRef = (element) => {
       this.svgRef = element;
     };
   }
 
   public componentDidMount() {
+    // Prevent scrolling on touch devices
+    // For iPad, listeners must be registered as non-passive in order to prevent scrolling
+    // Check touches.length to allow scrolling with 2 fingers
```

Comment:
If you have references that help with understanding these requirements it would be useful to post urls here. That way, if anything breaks at some point -- which it seems likely they will -- there will be something to go back to about how this was supposed to work originally.